package Algoritmos3.Excepciones;

public class CasilleroOcupadoException extends RuntimeException {
}
